packages/bpk-docs/src/pages/FormsPage/InputContainer.js

```diff
@@ -45,8 +46,10 @@ class InputContainer extends Component {
       overrideProps = {
         value: this.state.value,
         onChange: e => this.setState({ value: e.target.value }),
-        onClear: () => this.setState({ value: '' }),
       };
+      if (FormComponent === BpkInput) {
+        overrideProps.onClear = () => this.setState({ value: '' });
+      }
```

Comment:
Not related to this PR - we were getting an error from passing `onClear` to `BpkSelect` in the docs.